product_code/user_service/impl/src/test/scala/de/upb/cs/uc4/user/impl/UserServiceSpec.scala

```diff
@@ -172,7 +175,7 @@ class UserServiceSpec extends AsyncWordSpec
     "test topics, which" must {
       "publish a created user" in {
```

Comment:
This topic test works differently than the others, is it possible to change it such it does not fail in the case that other topics are already present?